CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/DatabaseConfigurationRequestHandler.swift

```diff
@@ -23,41 +23,41 @@ public class DatabaseConfigurationRequestHandler {
             
         case "databaseConfiguration_configure":
             let directory: String? = args.get(name:"directory")!
-            // let password: String? = args.get(name:"password")
-            // let encryptionKey: EncryptionKey? = password != nil ? EncryptionKey.password(password!) : nil
+            let password: String? = args.get(name:"password")
+            let encryptionKey: EncryptionKey? = password != nil ? EncryptionKey.password(password!) : nil
             let config = DatabaseConfiguration()
             if directory != nil {
                 config.directory = directory!
             }
-            /*
+
             #if COUCHBASE_ENTERPRISE
             if encryptionKey != nil {
                 config.encryptionKey = encryptionKey!
             }
-            #endif */
+            #endif
             return config
-            
+
         case "databaseConfiguration_getDirectory":
             let config: DatabaseConfiguration = args.get(name: "config")!
             return config.directory
-            
-        /*case "databaseConfiguration_getEncryptionKey":
+
+        case "databaseConfiguration_getEncryptionKey":
             let config: DatabaseConfiguration = args.get(name: "config")!
             return config.encryptionKey
-         */
+
         case "databaseConfiguration_setDirectory":
             let config: DatabaseConfiguration = args.get(name: "config")!
             let directory: String = args.get(name: "directory")!
             config.directory = directory
             return config
-            
-        /*case "databaseConfiguration_setEncryptionKey":
+
+        case "databaseConfiguration_setEncryptionKey":
             let config: DatabaseConfiguration = args.get(name: "config")!
             let password: String? = args.get(name:"password")!
-            //let encryptionKey: EncryptionKey? = password != nil ? EncryptionKey.password(password!) : nil
+            let encryptionKey: EncryptionKey? = EncryptionKey.password(password!)
```

Comment:
We might want to check for nil password here, else app might crash.

Reply:
I remember now that we are already handling nil value on server.swift. If it is nil, it replaces the value with space and never gets nil to the API methods. So we do not require this